torch/quantization/observer.py

```diff
@@ -199,6 +199,7 @@ def __init__(self, **kwargs):
         #  aten/src/ATen/native/quantized/cpu/qconv.cpp
         #  This is not the optimal choice for non x86 backends as
         #  lose a bit of precision for activations.
+        self.averaging_constant = kwargs.pop('averaging_constant', 0.01)
```

Comment:
nit: you don't need to do `pop`, just do `__init__(self, averaging_constant=0.01, **kwargs)` above